docs/user/install.md

```diff
@@ -0,0 +1,90 @@
+# Installing pyDARN 
+---
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0) [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/) [![GitHub version](https://badge.fury.io/gh/boennemann%2Fbadges.svg)](http://badge.fury.io/gh/boennemann%2Fbadges)
+
+## Prerequisites
+
+**python 3.6+**
+
+| Ubuntu      | OpenSuse       | Fedora        | OSX           |
+| ----------- | -------------- | ------------- | ------------- |
+| libyaml-dev | python3-PyYAML | libyaml-devel | Xcode/pip     |
+
+You can check your python version with  
+
+`$ python --version` or 
+`$ python3 --version`
+
+## Dependencies
+
+pydarn's setup will download the following dependencies: 
+
+- [NumPy](https://numpy.org/)
+- [matplotlib](https://matplotlib.org/)
+- [deepdish](https://deepdish.readthedocs.io/en/latest/api_io.html)
+- [pathlib2](https://docs.python.org/dev/library/pathlib.html)
+- [PyYAML](https://pyyaml.org/wiki/PyYAMLDocumentation)
+- [Cython](https://cython.org/)
+- [h5py](https://www.h5py.org/)
+
+## Virtual Environments
+It is recommended to install pyDARN in one of the suggested virtual environments if you have multiple python/pip 3 versions on your computer or, do not want to affect the main systems python libraries. 
+
+Please pick on of the following virtual environments. 
```

Comment:
"The following virtual environments have been tested by pyDARN developers:"

or at least replace "on" with "one"